src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -19,17 +22,33 @@ public class SearchConceptsQueryBuilder {
   private SearchConceptsQueryBuilder() {}
 
   public static String buildSearchConceptsQuery(
-      String domainId,
+      SnapshotBuilderDomainOption domainOption,
       String searchText,
       TableNameGenerator tableNameGenerator,
       CloudPlatformWrapper platform) {
     var conceptTablePointer = TablePointer.fromTableName("concept", tableNameGenerator);
+    var occurrenceTable = getOccurrenceTableFromDomain(domainOption.getId());
```

Comment:
I wonder if there's common code here and with other code that uses occurrence tables. It seems like there would be some commonality that could be refactored.

Reply:
In `CriteriaQueryBuilder.java` `getOccurenceTableFromDomain` is called in `generateFilter()`, since the scope is public, I'm thinking of just making this a helper function in another file? This is the second time this function is called and currently no commonality: 

It's called like this in `generateFilter` 

`OccurrenceTable occurrenceTable = getOccurrenceTableFromDomain(domainCriteria.getId());`
which very much resembles how we call it